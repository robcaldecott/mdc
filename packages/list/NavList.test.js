import React from "react";
import { NavList, NavListItem, NavListDivider } from ".";
import renderer from "react-test-renderer";

it("remders", () => {
  expect(
    renderer.create(
      <NavList className="custom">
        <NavListItem>Item 1</NavListItem>
        <NavListItem>Item 2</NavListItem>
        <NavListDivider />
        <NavListItem>Item 3</NavListItem>
      </NavList>
    )
  ).toMatchSnapshot();
});
