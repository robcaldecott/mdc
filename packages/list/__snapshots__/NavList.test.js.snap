// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`remders 1`] = `
<nav
  className="mdc-list custom"
>
  <a
    className="mdc-list-item"
  >
    Item 1
  </a>
  <a
    className="mdc-list-item"
  >
    Item 2
  </a>
  <hr
    className="mdc-list-divider"
    role="separator"
  />
  <a
    className="mdc-list-item"
  >
    Item 3
  </a>
</nav>
`;
